pkgs/development/tools/godot/generate-mono-deps.sh

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p coreutils dotnet-sdk nix jq
+# Script to generate the mono-deps.nix file.
+set -e
+
+# Check for dotnet2nix.
```

Comment:
Instead of this can we enter a shell with `dotnet2nix`, so that the user doesn't need to install it manually?